src/mixins/labels.js

```diff
@@ -124,6 +190,8 @@ export const LabelsMixin = (superclass) =>
          */
         toggleLabels() {
             this.areLabelsShown = !this.areLabelsShown;
+            this.#labels.forEach((label) => (label.visible = this.areLabelsShown));
+            this.#labels.visible = this.areLabelsShown;
```

Comment:
Our convention was to use `is...`, `are...` for booleans to make it clear - so maybe `visible` -> `areVisible`?

Reply:
Ohh... just found that the expression on line 194 is wrong, but the visible property is correct because it's a property of THREE.Object3d.

Reply:
https://threejs.org/docs/#api/en/core/Object3D.visible